Meziantou.Xunit.ParallelTestFramework/ParallelTestMethodRunner.cs

```diff
@@ -39,7 +39,7 @@ protected override async Task<RunSummary> RunTestCasesAsync()
         return summary;
     }
 
-    protected override async Task<RunSummary> RunTestCaseAsync(IXunitTestCase testCase)
+    protected new async Task<RunSummary> RunTestCaseAsync(IXunitTestCase testCase)
```

Comment:
Clever solution! Although I think I would probably change line 23 to check for the class level attributes instead.

```csharp
var disableParallelization = TestMethod.TestClass.Class.GetCustomAttributes(typeof(DisableParallelizationAttribute)).Any()
    || TestMethod.TestClass.Class.GetCustomAttributes(typeof(CollectionAttribute)).Any()
    || TestMethod.Method.GetCustomAttributes(typeof(DisableParallelizationAttribute)).Any()
    || TestMethod.Method.GetCustomAttributes(typeof(MemberDataAttribute)).Any(a => a.GetNamedArgument<bool>(nameof(MemberDataAttribute.DisableDiscoveryEnumeration)));
```

It's a couple extra lines of code this way, but I tend to avoid hiding inherited members with the `new` keyword since it can cause issues with inheritance.

Reply:
My first try was to use `TestMethod.Method.Type` but this throw an exception. `TestMethod.TestClass.Class` is what I was looking for. Thanks

Reply:
Thanks for the quick patch!